modules/nf-core/agat/spstatistics/tests/tags.yml

```diff
@@ -0,0 +1,2 @@
+agat/spstatistics:
+  - "modules/nf-core/agat/spstatistics/**"
```

Comment:
Should we remove the double quotes from the second line here? What's the better practice?

Reply:
I think the double quotes are meant to be....pinging @mashehu for confirmation

Reply:
not a strict rule, but would be good if we have the same style for all tags.yml and going by the [template for them](https://github.com/nf-core/tools/blob/master/nf_core/module-template/tests/tags.yml), please use double-quotes in tags.yml files.